training-maquetado/scss/_styles.scss

```diff
@@ -1,3 +1,5 @@
-.title {
-  color: red;
-}
+$text-big: 24px;
```

Comment:
i would call this file sth like `_sizes` and not styles

Reply:
I made a refactor in the next PR and most of the styles in `_styles.scss` were moved to another file `_texts.scss` which contains all the text sizes and the texts formats. Should I make this change also or we can consider the changes in the next PR?